source/overloading.tex

```diff
@@ -701,14 +701,12 @@
 \begin{ncsimplebnf}
 postfix-expression \terminal{(} \opt{expression-list} \terminal{)}
 \end{ncsimplebnf}
-if the \grammarterm{postfix-expression} denotes a set of overloaded functions and/or
-function templates, overload resolution is applied as specified in \ref{over.call.func}.
+overload resolution is applied as specified in \ref{over.call.func}.
```

Comment:
I don't think this change is correct; as discussed in Belfast, we don't want to perform overload resolution when the *postfix-expression* is a function pointer or similar. (This could be observably different, for example if there were a case where an implicit conversion sequence couldn't be formed despite implicit conversion being possible.)

Reply:
Fixed: "names at least one function or function template"